src/helpers/terminal_helpers.sh

```diff
@@ -113,6 +114,16 @@ function exit_with_status {
 }
 
 
+# Pipe to this function to indent output (2 spaces by default)
+function indent {
+  local count=2
+  [[ -n "$1" ]] && count=$1;
```

Comment:
I think its clearer to assign count to the argument first and if its empty then give it the default value.
Also just using a simple if statement makes it more readable to me.
```bash
local count=$1
if [ -z "$1" ]; then 
  count=2
fi 
```
You can inline the if you want to keep it shorter.